app/controllers/transactions_controller.rb

```diff
@@ -7,6 +7,12 @@ def create
     @transaction.save
   end
 
+  def create_transfer
+    @transfer = Transfer.new(transaction_params)
+    @transaction = Transaction.new(transaction_params)
```

Comment:
@Gataniel why do you need this Transaction Instance in this action? I think you can operate with Transfer instance, what you think?